src/software/ai/hl/stp/play/kickoff_enemy_play.cpp

```diff
@@ -82,22 +82,42 @@ void KickoffEnemyPlay::getNextTactics(TacticCoroutine::push_type &yield,
         std::make_shared<MoveTactic>(true), std::make_shared<MoveTactic>(true),
         std::make_shared<MoveTactic>(true)};
 
+    // created a new local variable Point at (zero,zero)
+    Point point = Point();
```

Comment:
nit: I would use `world.field().centrePoint()`

the centre should be at 0,0 but its good to get that from the field object

Reply:
Done